ot/gmm.py

```diff
@@ -0,0 +1,332 @@
+# -*- coding: utf-8 -*-
+"""
+Optimal transport for Gaussian Mixtures
+"""
+
+# Author: Eloi Tanguy <eloi.tanguy@u-paris>
+#         Remi Flamary <remi.flamary@polytehnique.edu>
+#         Julie Delon <julie.delon@math.cnrs.fr>
+#
+# License: MIT License
+
+from .backend import get_backend
+from .lp import emd2, emd
+import numpy as np
+from .lp import dist
+
+
+def gaussian_pdf(x, m, C):
```

Comment:
this function is not backed compatible add nx as parameter is if not note test for ackend please
```suggestion
def gaussian_pdf(x, m, C, nx=None):
```

Reply:
added backend compatibility (by adding `nx.det`)